R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -1,43 +1,91 @@
-# TODO: Document this guy
-
+#' @title Deephaven TableHandles
+#' @description Deephaven TableHandles are references to tables living on a Deephaven server. They provide an
+#' interface for interacting with tables on the server.
+#' 
+#' @usage NULL
+#' @format NULL
+#' @docType class
+#' 
+#' @section Methods
+#' 
+#' - `$bind_to_variable(name)`
+#' - `$to_arrow_record_batch_stream_reader()`
+#' - `$to_arrow_table()`
+#' - `$to_tibble()`
+#' - `$to_data_frame()`
+#'
+#' @examples
+#' 
+#' # connect to the Deephaven server running on "localhost:10000" using anonymous 'default' authentication
+#' client_options <- ClientOptions$new()
+#' client <- Client$new(target="localhost:10000", client_options=client_options)
+#' 
+#' # open a table that already exists on the server
+#' new_table_handle1 <- client$open_table("table_on_the_server")
+#' 
+#' # convert the Deephaven table to an R data frame
+#' new_data_frame <- new_table_handle1$to_data_frame()
+#' 
+#' # modify new dataframe in R
```

Comment:
Ironically, your comment about there being no inconsistencies is right before an inconsistency

Reply:
Yep...